code/__DEFINES/inventory.dm

```diff
@@ -156,7 +156,9 @@
 #define HEADCOVERSEYES		(1<<2)		// feel free to realloc these numbers for other purposes
 #define MASKCOVERSMOUTH		(1<<3)		// on other items, these are just for mask/head
 #define HEADCOVERSMOUTH		(1<<4)
-
+#define MASKPEPPERPROOF		(1<<5)	//protects against pepperspray 
+#define HEADPEPPERPROOF		(1<<6)
+#define EYESPEPPERPROOF		(1<<7)
```

Comment:
this should really just be 1 flag, i'm not sure why each clothing piece uses a unique bit.